src/pages/createMeeting/CreateMeeting.tsx

```diff
@@ -42,7 +45,16 @@ function CreateMeeting() {
 }
 
 export default CreateMeeting;
-
+const StepBar = styled.div<{ step: number }>`
+  width: ${({ step }) => `calc(100%* ${step}/6)`};
```

Comment:
```suggestion
  width: ${({ step }) => `calc(100%* ${step}/5)`};
```

develop머지 후에 총 step수를 5로 수정해주세요!!

Reply:
아 좋습니다~!